### PR TITLE
Add ability to set location

### PR DESCRIPTION
**Pull request by smarais**
### Issue

- [ ] My PR addresses the following Issue (If an issue exists in JIRA or Github)

### Description

- [ ] Here are some details about my PR, including example of usage.

### Licence

- [ ] My PR adds the needed licence header to every added file:

### Commits

- [ ] I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  2. Subject is limited to 50 characters
  3. Subject does not end with a period
  4. Subject uses the imperative mood ("add", not "adding")
  5. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

**Comment by smarais** [comment]
@mark-russell-bossanova here's the one-click-mlflow I was trying to deploy. It pulled in the "set location" fix that was sitting in the main fork...
My attempts to deploy (~3 attempts) had some arbitrary errors though that I think were unrelated to the change. I think it had to do with async components not spinning up in time. idk.

File: bin/set_various.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set -e
 
 PROJECT_ID=$(jq -r '.TF_VAR_project_id' ../vars.json)
-DOCKER_REPO="eu.gcr.io"
+DOCKER_REPO="gcr.io"
```
Comment (smarais) [inline bin/set_various.sh:6]: The gcr url (and in `vars_base` as well) is the eu component I was mentioning to look at for a possible EU component

Comment (mark-russell-bossanova) [reply bin/set_various.sh:6]: I made this change on my fork of this. I will test it and see if it works.

**Comment by smarais** [comment]
Closing, specific changes resolved in https://github.com/BossaNova/one-click-mlflow/pull/3

**Pull request closed without merging**